### PR TITLE
Update Library Samples

### PR DESCRIPTION
**Pull request by cartermp**
/cc @BillWagner @dend @richlander

### CODE REVIEW

File: samples/core-projects/libraries/new-library/test/LibraryTests/project.json [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
 	"frameworks": {
-		"dnxcore50":{
+		"netcoreapp1.0":{
 			"dependencies": {
-				"Library":"",
-				"System.Runtime":"4.0.0-rc1-*",
-				"xunit":"2.1.0",
-				"xunit.runner.dnx": "2.1.0-rc1-*"
+				"Library":{
+					"version": "1.0.0",
```
Comment (BillWagner) [inline samples/core-projects/libraries/new-library/test/LibraryTests/project.json]: I thought we recommended omitting the version element here for project reference targets. Did you mean a specific version?

File: samples/core-projects/libraries/new-library/src/Library/project.json [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+		"debugType": "portable"
 	},
 	"frameworks": {
-		"dotnet55":{
+		"netstandard1.6":{
 			"dependencies": {
-				"System.Runtime":"4.0.0-rc1-*"
+				"NETStandard.Library": "1.6.0",
+				"DependencyLibrary":{
+					"version": "1.0.0",
```
Comment (BillWagner) [inline samples/core-projects/libraries/new-library/src/Library/project.json]: I thought we recommended omitting the version element here for project reference targets. Did you mean a specific version?

File: samples/core-projects/libraries/new-library/test/LibraryTests/project.json (hunk 1 above)
Comment (cartermp) [inline samples/core-projects/libraries/new-library/test/LibraryTests/project.json]: Here's what happens when I omit the version:

screen shot 2016-07-07 at 10 58 01 am

Comment (BillWagner) [inline samples/core-projects/libraries/new-library/test/LibraryTests/project.json]: You have to run `dotnet restore` in the src/DependencyLibrary directory. dotnet restore doesn't follow dependent projects (yet). But, if you've run the restore for both projects, it works on both Windows and Mac.

/cc @blackdwarf  for more details.

Comment (cartermp) [inline samples/core-projects/libraries/new-library/test/LibraryTests/project.json]: Oh I'm stupid.  Never mind.

**Comment by mairaw** [comment]
Still working on this @cartermp?

**Comment by cartermp** [comment]
Yep, sorry, got lost in the mix

**Comment by cartermp** [comment]
Finished changes

**Comment by BillWagner** [comment]
Let's :shipit:!  It looks great to me.

**Pull request merged**